Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -1773,6 +1774,222 @@ public void When_the_actual_string_equivalent_to_the_expected_but_with_trailing_
 
         #endregion
 
+        #region ContainAll
+
+        [Fact]
+        public void When_null_collection_containall_should_throw_argument_exception()
```

Comment:
Please don't repeat the name of a method or member in the name of the test case. See point 11 of http://www.continuousimprover.com/2015/11/12-tips-to-write-unit-tests-that-dont.html